lib/c-api/examples/wasmer-capi-examples-runner/src/lib.rs

```diff
@@ -292,7 +292,7 @@ fn test_run() {
             println!("setting current dir = {exe_dir}");
             let output = command
                 .output()
-                .expect(&format!("failed to run {command:#?}"));
+                .unwrap_or_else(|_| panic!("failed to run {command:#?}"));
```

Comment:
We should use the previous code

Reply:
@syrusakbary this is a clippy autofix done by `clippy --fix --all`, it does exactly the same thing, but it doesn't construct a string in the "okay" case. Same for all the other issues.